mobilecoind/clients/python/cli/README.md

```diff
@@ -0,0 +1,114 @@
+## Python CLI tools for `mobilecoind`
+
+### Getting your Public Address
+
+Every 32-byte Master Key defines a unique MobileCoin account. You can share the corresponding public address with other users by sending them the base-58 encoded "Address Code" or the "Address URL" generated by this CLI tool. The "Address URL" can be conveniently embedded as the payload of a QR code.
+
+Note that this command creates a monitor for the provided or generated Master Key that will remain active until removed.
+
+Help:
+```
```

Comment:
I would suggest not including the help in the README, so that it's easier to maintain the README, and users can run help as needed.